doc/15-troubleshooting.md

```diff
@@ -1628,22 +1628,19 @@ In order to solve this problem, remove the mentioned files from `zones.d` and us
 of syncing plugin binaries to your satellites and agents.
 
 
-#### Zones in Zones doesn't work <a id="troubleshooting-cluster-config-zones-in-zones"></a>
+#### Zones in Zones <a id="troubleshooting-cluster-config-zones-in-zones"></a>
 
-The cluster config sync works in the way that configuration
-put into `/etc/icinga2/zones.d` only is included when configured
-outside in `/etc/icinga2/zones.conf`.
+The cluster config sync works in the way that configuration put into `/etc/icinga2/zones.d` only is included when
+configured either in `/etc/icinga2/zones.d/localZoneName` or outside in `/etc/icinga2/zones.conf`.
 
-If you for example create a "Zone Inception" with defining the
-`satellite` zone in `zones.d/master`, the config compiler does not
-re-run and include this zone config recursively from `zones.d/satellite`.
+If you for example add some configs in to `zones.d/satellite` and forgot to create a "Zone Inception" with defining
+the `satellite` zone in `zones.d/master` or outside in `/etc/icinga2/zones.conf`, the config compiler will not include
+this config from `zones.d/satellite` zone directory.
 
 Since v2.11, the config compiler is only including directories where a
-zone has been configured. Otherwise it would include renamed old zones,
+zone has been configured. Otherwise, it would include renamed old zones,
 broken zones, etc. and those long-lasting bugs have been now fixed.
 
-A more concrete example: Masters and Satellites still need to know the Zone hierarchy outside of `zones.d` synced configuration.
-
 **Doesn't work**
```

Comment:
Not sure if this is still good wording now. I mean the following config now won't work because it just misses a definition of the `satellite` zone altogether. Maybe make all the following sections working examples and point out the details that make them work?